pkg/monitoring/rules/recordingrules/operator.go

```diff
@@ -29,6 +29,6 @@ var operatorRecordingRules = []operatorrules.RecordingRule{
 			Help: "Monitors resources for potential problems",
 		},
 		MetricType: operatormetrics.GaugeType,
-		Expr:       intstr.FromString(`sum by (container, node, reason)(kubevirt_memory_delta_from_requested_bytes)`),
+		Expr:       intstr.FromString(`sum by (container, reason)(kubevirt_memory_delta_from_requested_bytes)`),
```

Comment:
fine by me, but why do we need to remove the node label? cardinality?

Reply:
Yes, cardinality, they say the node names may vary over time which may add too many series to telemeter

Reply:
makes sense